### ISSUE TITLE
Per-game setting profiles

### ISSUE THREAD

**Issue by NGnius** [NGnius/PowerTools#8]
**Describe what you'd like to be able to do**
Choose & apply settings based on the currently-running game, instead of globally

**Describe alternatives you've considered**
Manually changing the settings after each game is launched. This is tedious and requires a good memory/notepad.

**Additional context**
Investigation is required to find out how Steam handles it's per-game configs for non-Steam apps. For steam games, there's probably sufficient information in `~/.steam/registry.vdf`

**Comment by notimp** [comment]
After reboot Power Tools is disabled and should remain so. Please document, how you will enable profile loading (triggered?) by game (loading?) as well. :) Thank you.

**Comment by NGnius** [comment]
When v0.5.0 is released next week there will be a toggle to enable keeping your PowerTools settings after a reboot. Per-game settings will be an extension of this persistence functionality.

I do plan on documenting how detection of games is done, and I will document how & where settings are saved if/when I find a good way to load settings when a game is launched (in v0.5.0 the single settings file will only be loaded when PowerTools is loaded -- at boot -- so this does not apply).

**Comment by NGnius** [comment]
Code for detecting game launch
```javascript
SteamClient.Apps.RegisterForGameActionStart((actionType, data) => {
  console.log("start", await appStore.GetAppOverviewByGameID(data[1])))
})
```

**Comment by NGnius** [comment]
Implemented in 7158ac3596f9e8ec93e05b87bc853973540e695b, still need to do some testing to convince myself it's less buggy than v0.5.0
